src/main.rs

```diff
@@ -230,6 +230,21 @@ async fn main() -> Result<()> {
                     .await?;
                 "".to_string()
             }
+            #[cfg(feature = "plugin-vc-zkp-bbs")]
+            ("create_self_issued_credential", Some(sub_m)) => {
```

Comment:
Please check and add the subcommand to helper =>
fn add_subcommand_helper<'a>(app: App<'a, 'a>) -> Result<App<'a, 'a>>

Reply:
Done, thanks